cmd/system-probe/probe.go

```diff
@@ -40,8 +39,11 @@ func CreateSystemProbe(cfg *config.AgentConfig) (*SystemProbe, error) {
 	nt := &SystemProbe{}
 
 	// Checking whether the current OS + kernel version is supported by the tracer
-	if nt.supported, err = ebpf.IsTracerSupportedByOS(cfg.ExcludedBPFLinuxVersions); err != nil {
+	if supported, err := ebpf.IsTracerSupportedByOS(cfg.ExcludedBPFLinuxVersions); err != nil {
 		return nil, fmt.Errorf("%s: %s", ErrTracerUnsupported, err)
+	} else if !supported {
+		platform, _ := util.GetPlatform()
+		return nil, fmt.Errorf("%s: please check your current kernel version: %s", ErrTracerUnsupported, platform)
```

Comment:
Is there other logging now that indicates the problem? Just saying to check the kernel version isn't that useful

Reply:
Not really but if the system-probe is not supported it's probably because of a too old kernel version no ?